.circleci/config.yml

```diff
@@ -6,14 +6,32 @@ orbs:
 
 jobs:
   run-specs-with-postgres:
-    executor: solidusio_extensions/postgres
+    docker:
+      - image: cimg/ruby:2.7-browsers
+        environment:
+          DB: postgresql
+          PGHOST: 127.0.0.1
+          PGUSER: postgres
+          RAILS_ENV: test
+      - image: cimg/postgres:14.2
+        environment:
+          POSTGRES_USER: postgres
     steps:
       - checkout
       - solidusio_extensions/run-tests-solidus-older
       - solidusio_extensions/run-tests-solidus-current
       - solidusio_extensions/store-test-results
   run-specs-with-master:
-    executor: solidusio_extensions/postgres
+    docker:
+      - image: cimg/ruby:2.7-browsers
+        environment:
+          DB: postgresql
+          PGHOST: 127.0.0.1
+          PGUSER: postgres
+          RAILS_ENV: test
+      - image: cimg/postgres:14.2
+        environment:
+          POSTGRES_USER: postgres
```

Comment:
Could we dry this up by defining this yaml in our own executor block as per this documentation https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-executors